### ISSUE TITLE
Missmatch between default rules documentation?

### ISSUE THREAD

**Issue by molant** [dequelabs/axe-core#907]
Hello all,

in [sonarwhal](https://sonarwhal.com) we are using aXe's default options. Per [the documentation](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter) we thought that we were running only the WCAG 2.0 Level A and Level AA rules:

> The default operation for axe.run is to run all WCAG 2.0 Level A and Level AA rules. If certain rules should be disabled from being run, specify `options` as:

@aardrian brought us to at attention that this might not be the case in [this issue](https://github.com/sonarwhal/sonarwhal/issues/1087) and looking into the [rule descriptions](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) it looks like there are more rules running by default.

Can you please confirm what set of rules is being run?

Thanks!

**Comment by aardrian** [comment]
> That begs a larger question, should best practices be turned on by default?

For those who may be using it for assessing legal compliance, _must_ statements (in particular) can imply a legal requirement and drain resources from addressing issues that are truly legal risks (real WCAG issues).

Since not all tools that lean on default settings may surface those you have flagged as best practices, it might be safer to leave them off by default.

**Comment by WilcoFiers** [comment]
Good point about the docs. I do think Axe-core should run with best practices enabled by default. The most common use of Axe-core is developers using it during feature development. That is where best practices should be used. For compliance best practices should be turned off.

**Comment by darekkay** [comment]
Just to be sure - this decision was changed with `axe-core` 4.0, correct? From the [new docs](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md)

> The default operation for axe.run is to run all rules except for rules with the "experimental" tag

**Comment by straker** [comment]
I don't think anything has changed about which rules run in 4.0. Since at least 3.0 axe runs all rules except experimental, and that continues through 3.0.

The only axe tool that runs experimental rules is axe-coconut.
